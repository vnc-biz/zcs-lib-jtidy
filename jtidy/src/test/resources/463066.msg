<?xml version="1.0" encoding="ISO-8859-1"?>
<!-- expected messages for test 463066-->
<messages>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[
Tidy (vers 4th August 2004) Parsing "InputStream"
]]></text>
  </message>
  <message>
    <code>29</code>
    <level>2</level>
    <line>1</line>
    <column>11</column>
    <text><![CDATA[adjacent hyphens within comment]]></text>
  </message>
  <message>
    <code>48</code>
    <level>2</level>
    <line>3</line>
    <column>1</column>
    <text><![CDATA[unknown attribute "xmlns:dt"]]></text>
  </message>
  <message>
    <code>48</code>
    <level>2</level>
    <line>3</line>
    <column>1</column>
    <text><![CDATA[unknown attribute "xmlns:w"]]></text>
  </message>
  <message>
    <code>48</code>
    <level>2</level>
    <line>3</line>
    <column>1</column>
    <text><![CDATA[unknown attribute "xmlns:o"]]></text>
  </message>
  <message>
    <code>44</code>
    <level>2</level>
    <line>3</line>
    <column>1</column>
    <text><![CDATA[missing <!DOCTYPE> declaration]]></text>
  </message>
  <message>
    <code>49</code>
    <level>2</level>
    <line>79</line>
    <column>1</column>
    <text><![CDATA[<style> lacks "type" attribute]]></text>
  </message>
  <message>
    <code>21</code>
    <level>2</level>
    <line>539</line>
    <column>62</column>
    <text><![CDATA[<o:p> is not approved by W3C]]></text>
  </message>
  <message>
    <code>23</code>
    <level>2</level>
    <line>539</line>
    <column>63</column>
    <text><![CDATA[trimming empty <o:p>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>20</code>
    <level>2</level>
    <line>553</line>
    <column>1</column>
    <text><![CDATA[replacing element <p> by <li>]]></text>
  </message>
  <message>
    <code>111</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Document content looks like HTML proprietary]]></text>
  </message>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[20 warnings, no errors were found!
]]></text>
  </message>
</messages>
