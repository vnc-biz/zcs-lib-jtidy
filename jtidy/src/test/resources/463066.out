<html>
<head>
<title>[ #463066 ] CleanWord2000 misses mso-list bullets</title>
</head>
<body>
<div class="Section1">
<p>Test 1</p>
<ol start="1" type="1">
<li>Here</li>
<li>We</li>
<li>Go</li>
<li>Again</li>
</ol>
&nbsp;
<ol start="1" type="i">
<li>Ok</li>
<li>That</li>
<li>Worked</li>
</ol>
&nbsp;
<ul>
<li>But</li>
<li>It</li>
<li>Does</li>
<li>Not</li>
<li>Work</li>
<li>With</li>
<li>Bullet</li>
<li>Points</li>
<li>Now</li>
</ul>
&nbsp;
<ul>
<li>It</li>
<li>Is</li>
<li>Working</li>
</ul>
&nbsp;
<ul type="disc">
<li>Try</li>
<li>It</li>
<li>Again</li>
</ul>
&nbsp;
<ul type="disc">
<li>Do</li>
<li>It</li>
<li>Again</li>
<li>And</li>
</ul>
<p><o:p></o:p></p>
</div>
</body>
</html>
